app/emoji-custom/client/admin/adminEmoji.js

```diff
@@ -31,7 +31,10 @@ Template.adminEmoji.helpers({
 	hasMore() {
 		if (Template.instance().limit != null) {
 			if (typeof Template.instance().customemoji === 'function') {
-				return Template.instance().limit.get() === Template.instance().customemoji().length;
+				return (
+					Template.instance().limit.get()
+					=== Template.instance().customemoji().length
```

Comment:
Please revert this change, makes it hard to read...

Reply:
sorry, autolint 😞

Reply:
Yikes, that's an aggressive one 😮